Crash report 4.1 - OOBException accessing Cursor

Need to check if [moveToFirst](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/media/MediaAddFragment.java#L201) returns true before accessing data from it.
```
java.lang.RuntimeException: Failure delivering result ResultInfo{who=android:fragment:1, request=2000, result=-1, data=Intent { dat=content://media/external/images/media/39916 flg=0x1 }} to activity {org.wordpress.android/org.wordpress.android.ui.media.MediaBrowserActivity}: android.database.CursorIndexOutOfBoundsException: Index 0 requested, with a size of 0
       at android.app.ActivityThread.deliverResults(ActivityThread.java:3568)
       at android.app.ActivityThread.handleSendResult(ActivityThread.java:3611)
       at android.app.ActivityThread.access$1400(ActivityThread.java:147)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1345)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:135)
       at android.app.ActivityThread.main(ActivityThread.java:5254)
       at java.lang.reflect.Method.invoke(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:372)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:898)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:693)
Caused by: android.database.CursorIndexOutOfBoundsException: Index 0 requested, with a size of 0
       at android.database.AbstractCursor.checkPosition(AbstractCursor.java:426)
       at android.database.AbstractWindowedCursor.checkPosition(AbstractWindowedCursor.java:136)
       at android.database.AbstractWindowedCursor.getString(AbstractWindowedCursor.java:50)
       at android.database.CursorWrapper.getString(CursorWrapper.java:114)
       at org.wordpress.android.ui.media.MediaAddFragment.getRealPathFromContentURI(MediaAddFragment.java:202)
       at org.wordpress.android.ui.media.MediaAddFragment.getRealPathFromURI(MediaAddFragment.java:175)
       at org.wordpress.android.ui.media.MediaAddFragment.fetchMedia(MediaAddFragment.java:167)
       at org.wordpress.android.ui.media.MediaAddFragment.onActivityResult(MediaAddFragment.java:142)
       at android.app.Activity.dispatchActivityResult(Activity.java:6139)
       at android.app.ActivityThread.deliverResults(ActivityThread.java:3564)
       at android.app.ActivityThread.handleSendResult(ActivityThread.java:3611)
       at android.app.ActivityThread.access$1400(ActivityThread.java:147)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1345)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:135)
       at android.app.ActivityThread.main(ActivityThread.java:5254)
       at java.lang.reflect.Method.invoke(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:372)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:898)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:693)
```

--- a/WordPress/src/main/java/org/wordpress/android/ui/media/MediaAddFragment.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/media/MediaAddFragment.java
@@ -198,11 +198,13 @@ public class MediaAddFragment extends Fragment implements LaunchCameraCallback {
             return null;
         }
 
-        if (!cursor.moveToFirst()) {
-            return null;
+        String path;
+        if (cursor.moveToFirst()) {
+            path = cursor.getString(column_index);
+        } else {
+            path = null;
         }
 
-        String path = cursor.getString(column_index);
         cursor.close();
         return path;
     }

--- a/WordPress/src/main/java/org/wordpress/android/ui/media/MediaAddFragment.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/media/MediaAddFragment.java
@@ -198,7 +198,10 @@ public class MediaAddFragment extends Fragment implements LaunchCameraCallback {
             return null;
         }
 
-        cursor.moveToFirst();
+        if (!cursor.moveToFirst()) {
+            return null;
+        }
+
         String path = cursor.getString(column_index);
         cursor.close();
         return path;

